wa/framework/target/info.py

```diff
@@ -53,9 +53,9 @@ def kernel_version_from_pod(pod):
 
 def kernel_config_from_pod(pod):
     config = KernelConfig('')
-    config._config = pod['kernel_config']
+    config.typed_config._config = pod['kernel_config']
     lines = []
-    for key, value in config._config.items():
+    for key, value in config.iteritems():
```

Comment:
Should be `items()` to be compatible with Python 3

Reply:
This is actual a method of a `KernelConfg` object rather than a python built in.
**https://github.com/ARM-software/devlib/blob/master/devlib/target.py#L1998**

Would it make sense to add an alias to `KernelConfig` to avoid confusion?

Reply:
Ah, my mistake. Yes, and alias might be a good idea.

Reply:
Updated to use aliased method.